src/ui/flutter_app/lib/widgets/game_page.dart

```diff
@@ -1113,30 +1126,36 @@ class _GamePageState extends State<GamePage>
             ),
           ),
           actions: <Widget>[
-            end > 0
-                ? TextButton(
-                    child: Text(S.of(context).rollback,
-                        style: AppTheme.moveHistoryTextStyle),
-                    onPressed: () async {
-                      int selectValue = await showPickerNumber(
-                        context,
-                        1,
-                        end,
-                        1,
-                        S.of(context).moves,
-                      );
-
-                      if (selectValue != 0) {
-                        onTakeBackNButtonPressed(selectValue);
-                      }
-                    },
-                  )
-                : TextButton(
-                    child: Text(""),
-                    onPressed: () => Navigator.of(context).pop()),
+            if (end > 0)
```

Comment:
This paragraph is also without {}, I don't know if it conforms to the specification.